charts/kyverno/templates/NOTES.txt

```diff
@@ -36,3 +36,7 @@ The following components have been installed in your cluster:
 {{- with .Values.config.matchConditions }}
 ⚠️  WARNING: Match conditions require a Kubernetes 1.27+ cluster with `AdmissionWebhookMatchConditions` feature gate enabled.
 {{- end }}
+
+{{- with .Values.features.generateValidatingAdmissionPolicy.enabled }}
+⚠️  WARNING: Generate validating admission policy require a Kubernetes 1.26+ cluster with `ValidatingAdmissionPolicy` feature gate enabled.
```

Comment:
Suggestion:

"Generating ValidatingAdmissionPolicy requires Kubernetes 1.26+ with the `ValidatingAdmissionPolicy` feature gate and `admissionregistration.k8s.io` API group enabled."

Reply:
Done.